### ISSUE TITLE
Report comments such as // ================= as own rule but not MiKo_2300

### ISSUE THREAD

**Issue by RalfKoban** [RalfKoban/MiKo-Analyzers#967]
Comments used for separations, such as
```c#
// ==========
// **********
// ----------
// ##########
```

should be reported as unnecessary and should be removed.
However, that should be done by a specific rule but not `MiKo_2300`.
